R/cbc_solve.R

```diff
@@ -1,3 +1,5 @@
+#' @import assertthat
```

Comment:
Could you please use `@imprtFrom` and only import the functions actually use? `assert_that`, `is.numeric`